vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -1,109 +1,161 @@
+2. Click `Add Role`
+3. Fill in the `Role Name` with `conference-admin`
+4. Click `Save`
+5. Repeat these steps to create the `conference-user` role
 
-Now we need to map this role to our user.
+> Note: The `internal` client is configured by default in the Spring Boot `application.yml`.
 
-8. Go to `Users → View all users → admin → Role Mappings`
-9. Select `internal` for the `Client Roles` and then move `conference-user` from `Available Roles` to `Assigned Roles`
-10. Go back to the MFE and you should now see the full list of Conferences.
+### Step 6: Map the `conference-user` role to the admin user
 
-We've now successfully secured the `getAllConferences` API but we have more to do. The admin user was granted just the `conference-user` role but still has access to delete Conferences. We need to lock that down.
+To secure the `getAllConferences` API:
 
-11. Go back into the `ConferenceResource.java` file and add this annotation to the `deleteConference` method:
+1. Go to `Users` → `View all users` → `admin` → `Role Mappings`
+2. Select `internal` for the `Client Roles` 
+3. Move `conference-user` from `Available Roles` to `Assigned Roles`
+4. Return to the MFE to confirm you see the full list of Conferences
 
+### Step 7: Restrict the ability to delete Conferences
+
+The `conference-user` role grants the admin user permission to delete Conferences. To restrict the delete method to the `conference-admin` role:
+
+1. Go to the `src/main/java/com/<ORG>/<NAME>.web.rest` directory
+2. Open `ConferenceResource.java` 
+3. Modify the `deleteConference` method by adding the following annotation:
 ```
     @PreAuthorize("hasAuthority('conference-admin')")
-    public ResponseEntity<Void> deleteConference(@PathVariable Long id) {
+    public ResponseEntity<Void> deleteConference(@PathVariable Long ID) {
 ```
-Here we're restricting the delete method to only the `conference-admin` role.
 
-12. Restart the microservice. By default this will include rebuilding any changed source files.
-13. Once the microservice is available, go back to the MFE and try deleting one of the Conferences in the list. You should be able to attempt the delete in the UI but you'll get a 403 error in the browser console and an error like this in the service logs:
+To verify that a user without the `conference-admin` role is unable to call the delete API:
+
+1. Restart the microservice. By default this includes rebuilding any changed source files.
+2. Once the microservice is available, return to the MFE and try deleting one of the Conferences in the list 
+3. Verify that attempting to delete via the UI generates a `403 error` in the browser console and an error in the service logs similiar to the following:
 ```
 2021-03-22 15:56:16.205  WARN 3208 --- [  XNIO-1 task-3] o.z.problem.spring.common.AdviceTraits   : Forbidden: Access is denied
 ```
-That's exactly what we wanted! This demonstrates that a user without `conference-admin` is unable to call the delete API.
```

Comment:
Maybe keep the "This demonstrates..." statement.

Reply:
added back but want to discuss

Reply:
On a second read, I think it can be dropped since step#3 leads with "Verify that..."

Reply:
On a second read, I think it can be dropped since step#3 leads with "Verify that..."